client/src/utils/fetchUtils.ts

```diff
@@ -15,8 +15,15 @@ export async function postRegistrertSykmelding(
     oppgaveid: number,
     enhet: string,
     sykmelding: RegistrertSykmelding,
+    isFerdigstilt: boolean,
 ): Promise<void> {
-    const res = await fetch(`backend/api/v1/oppgave/${oppgaveid}/send`, {
+    let url: string;
+    if (isFerdigstilt) {
+        url = `backend/api/v1/oppgave/${oppgaveid}/endre`;
+    } else {
+        url = `backend/api/v1/oppgave/${oppgaveid}/send`;
+    }
```

Comment:
her kan du kanskje spandere på deg en ternary uten at det blir for vanskelig kode, så sparer man seg en `let`.

```ts
const url = isFerdigStilt ? `backend/api/v1/oppgave/${oppgaveid}/endre` : `backend/api/v1/oppgave/${oppgaveid}/send`;
```

Reply:
God idé!